.devcontainer/develop-livebook/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Update the VARIANT arg in docker-compose.yml to pick an Elixir version: 1.9, 1.10, 1.10.4
+ARG VARIANT="1.13.4"
+FROM elixir:${VARIANT}
```

Comment:
Would you be cool with
```Dockerfile
ARG ELIXIR_VERSION="1.13.2"
ARG ERLANG_VERSION="24.1.7"
ARG DEBIAN_VERSION="20210902"
FROM hexpm/elixir:${ELIXIR_VERSION}-erlang-${ERLANG_VERSION}-debian-bullseye-${DEBIAN_VERSION}-slim
```

Use case: If a developer wants to try bumping the Elixir/Erlang/Debian version to see what breaking changes (if any) there are to Livebook.

Thoughts? 😄

Reply:
Does GH spaces have UI for configuring those?

Reply:
Unfortunately no. You _could_ create a separate `devcontainer.json` that automatically uses the latest version of each, but too many containers to choose from would be overwhelming.

All this would do is allow users to potentially fork `livebook`, modify the `docker-compose.yml` file with new versions, and start a codespace from there.

Reply:
I see, in that case the could as well change the Dockerfile. The good thing about having the full image name is that it's easier to find and replace it when bumping the main Dockerfile we have. That said, either works for me, so whatever you prefer :)

Reply:
I decided to hardcode it and refer back to the root Dockerfile!